### PR TITLE
Implemented additional styling options for markdown output

### PR DESCRIPTION
**Pull request by aaAldinaaa**
Added logic for more styling options:

- changed default style to 'red_green' to show deletions in red, additions in green
- added element attributes (class attribute for now) to be applied to the <span> tags for possible future HTML output implementation

### CODE REVIEW

File: redlines/redlines.py [hunk 1]
```diff
@@ -133,22 +133,43 @@ def opcodes(self) -> list[tuple[str, int, int, int, int]]:
...
         if self.options.get("markdown_style"):
             style = self.options["markdown_style"]
 
         if style == "none":
             md_styles = {"ins": ("ins", "ins"), "del": ("del", "del")}
-        elif "red":
+        elif style == 'red_green':
+            md_styles = {
+                "ins": (
+                    f"span {elem_attributes['ins']['class']} style='color:green;font-weight:700;'",
```
Comment (inteli5) [inline redlines/redlines.py:152]: should here be f"span class='{elem_attributes['ins']['class']}' style='color:green;font-weight:700;'", ?

File: redlines/redlines.py [hunk 2]
```diff
@@ -133,22 +133,43 @@ def opcodes(self) -> list[tuple[str, int, int, int, int]]:
...
+                "del": (
+                    f"span {elem_attributes['del']['class']} style='color:red;font-weight:700;text-decoration:line-through;'",
+                    'span'
+                )
+            }
+        elif style == "red":
             md_styles = {
-                "ins": ('span style="color:red;font-weight:700;"', "span"),
+                "ins": (
+                    f"span {elem_attributes['ins']['class']} style='color:red;font-weight:700;'",
```
Comment (inteli5) [inline redlines/redlines.py:163]: Is this a custom element attribute for the <span>? The result will look like `<span inserted style='color:red;font-weight:700;'>text</span>`? I am not familiar with this. Is it valid in HTML? If it is valid, what is the functionality of this element attribute? thank you.

Comment (aaAldinaaa) [reply redlines/redlines.py]: Thank you, I just pushed a change to the branch that fixes the initial attribute implementation. It produces the correct syntax now: `<span class='inserted' style='color:red;font-weight:700;'>text</span>`

The attributes are meant to serve potential future implementations of HTML output, and to help with their styling respectively.

**Comment by houfu** [comment]
Hi, could we have more tests, for example, what is the expected_md for `red_green` at  https://github.com/houfu/redlines/blob/102b396e9e46eca4257c3fffdfd32fb917d1ab67/tests/test_redline.py#L87?

How does the element attribute help by the way? (How do you see this implemented)?

Thanks for trying to contribute!

**Comment by inteli5** [comment]
Hi, @houfu  what is a good email to reach you? I may ask you some small questions from time to time, but it is not as big as an issue. So email would be convenient. My email is zhoubo97@gmail.com

**Comment by cauemello** [comment]
Hello! Looking forward to this PR.

**Comment by houfu** [comment]
Think I'll merge this first and fix anything necessary in another commit.
